assignments/arrays.js

```diff
@@ -75,30 +75,51 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+console.log(`Car 33 is a ${inventory[32].car_model} ${inventory[32].car_make} ${inventory[32].car_year}`);
```

Comment:
You can improve your solution here, by using for loop, in large datasets, and another case of unsorted ids.